content/docs/manuals/networking-internals/overlay-network.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "Overlay Network"
+summary: ""
+draft: false
+weight: 311
+toc: true
+seo:
+  title: "" # custom title (optional)
+  description: "" # custom description (recommended)
+  canonical: "" # custom canonical URL (optional)
+  noindex: false # false (default) or true
+---
+
+We start by explaining the different network abstraction layers in place and then continue with the semantic addressing.
```

Comment:
Since semantic addressing is not detaild here, can you link semantic addressind to the load balancing section?

Reply:
Good point, done!